### PR TITLE
Fix TLS configurations

### PR DESCRIPTION
**Pull request by pillip8282**
Fix TLS configurations

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [f6d29f4a948c166f1ae8b5e05c344c2e6cc61d38] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [f6d29f4a948c166f1ae8b5e05c344c2e6cc61d38] - Code Rule Check OK.

File: external/include/mbedtls/config.h [hunk 1]
```diff
@@ -548,10 +548,10 @@
  *
  * Enable padding modes in the cipher layer.
  */
-#define MBEDTLS_CIPHER_PADDING_PKCS7
-#define MBEDTLS_CIPHER_PADDING_ONE_AND_ZEROS
-#define MBEDTLS_CIPHER_PADDING_ZEROS_AND_LEN
-#define MBEDTLS_CIPHER_PADDING_ZEROS
+//#define MBEDTLS_CIPHER_PADDING_PKCS7
+//#define MBEDTLS_CIPHER_PADDING_ONE_AND_ZEROS
```
Comment (NilabhShant) [inline external/include/mbedtls/config.h]: Please remove these lines if not needed.

File: external/include/mbedtls/config.h [hunk 2]
```diff
@@ -1172,7 +1172,7 @@
  *
  * Comment this macro to disable support for Encrypt-then-MAC
  */
-#define MBEDTLS_SSL_ENCRYPT_THEN_MAC
+// #define MBEDTLS_SSL_ENCRYPT_THEN_MAC // artik
```
Comment (NilabhShant) [inline external/include/mbedtls/config.h]: same as above.

**Comment by seinfra** [comment]
Target : [589771c7138eed22398aa128c0547a023d820df9] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [589771c7138eed22398aa128c0547a023d820df9] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [9127e316fc875106d6a3a5bdbc1c2789628121e0] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [9127e316fc875106d6a3a5bdbc1c2789628121e0] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [2735976e6f91f55fc017cec46d0772e928631aff] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [2735976e6f91f55fc017cec46d0772e928631aff] - Code Rule Check OK.

**Pull request closed without merging**